DeploymentJenkinsfile

```diff
@@ -0,0 +1,170 @@
+pipeline {
```

Comment:
this is separate job, which deploys microservices from  `platform_deployment.yml`